src/get_functions.jl

```diff
@@ -0,0 +1,48 @@
+"""
+    get_origin( vtk_file )
+
+Retrieve the vector of coordinates of the origin of a uniform grid from the given [`VTKFile`](@ref) file.
+
+See also: [`VTKFile`](@ref)
+"""
+function get_origin(vtk_file)
+  # check imagedata
+  if !(vtk_file.file_type == "ImageData")
```

Comment:
```suggestion
  if vtk_file.file_type != "ImageData"
```
easier to comprehend